lib/ExpensiMark.js

```diff
@@ -9,6 +9,42 @@ const MARKDOWN_IMAGE_REGEX = new RegExp(`\\!${MARKDOWN_LINK}`, 'gi');
 
 const SLACK_SPAN_NEW_LINE_TAG = '<span class="c-mrkdwn__br" data-stringify-type="paragraph-break" style="box-sizing: inherit; display: block; height: unset;"></span>';
 
+const ASTERISK_ITALIC = '*';
+const UNDERSCORE_ITALIC = '_';
+const ASTERISK_BOLD = '**';
+const UNDERSCORE_BOLD = '__';
+const ESCAPED_UNDERSCORE = '\\_';
+const ESCAPED_ASTERISK = '\\*';
+const ASTERISK_PLACEHOLDER_REGEXP = /ASTERISKPLACEHOLDER/gm;
+const UNDERSCORE_PLACEHOLDER_REGEXP = /UNDERSCOREPLACEHOLDER/gm;
+const UNDERSCORE_BOLD_PLACEHOLDER_REGEXP = /UNDERSCOREBOLDPLACEHOLDER/gm;
+const ASTERISK_BOLD_PLACEHOLDER_REGEXP = /ASTERISKBOLDPLACEHOLDER/gm;
+const UNDERSCORE_ITALIC_PLACEHOLDER_REGEXP = /UNDERSCOREITALICPLACEHOLDER/gm;
+const ASTERISK_ITALIC_PLACEHOLDER_REGEXP = /ASTERISKITALICPLACEHOLDER/gm;
+
+
+const ESCAPED_ASTERISK_REGEXP = /\\\*/g;
+
+const ESCAPED_UNDERSCORE_REGEXP = /\\_/g;
+const UNDERSCORE_BOLD_REGEXP = /(__)(.*?)(__)/g;
+const ASTERISK_BOLD_REGEXP = /(\*\*)(.*?)(\*\*)/g;
+const UNDERSCORE_ITALIC_REGEXP = /(_)(.*?)(_)/g;
+const ASTERISK_ITALIC_REGEXP = /(\*)(.*?)(\*)/g;
+
+const HYPERLINK = /^\[([^[]+)\]\(([^)]+)\)/;
```

Comment:
Some more consistent whitespace: 

```suggestion
const ASTERISK_ITALIC_PLACEHOLDER_REGEXP = /ASTERISKITALICPLACEHOLDER/gm;

const ESCAPED_ASTERISK_REGEXP = /\\\*/g;
const ESCAPED_UNDERSCORE_REGEXP = /\\_/g;
const UNDERSCORE_BOLD_REGEXP = /(__)(.*?)(__)/g;
const ASTERISK_BOLD_REGEXP = /(\*\*)(.*?)(\*\*)/g;
const UNDERSCORE_ITALIC_REGEXP = /(_)(.*?)(_)/g;
const ASTERISK_ITALIC_REGEXP = /(\*)(.*?)(\*)/g;
const HYPERLINK = /^\[([^[]+)\]\(([^)]+)\)/;
```